.github/workflows/test-component-library-vizro-core.yml

```
@@ -0,0 +1,66 @@
+name: Integration tests Component Library
+
+defaults:
+  run:
+    working-directory: vizro-core
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
+
+env:
+  PYTHONUNBUFFERED: 1
+  FORCE_COLOR: 1
+
+jobs:
+  test-component-library-vizro-core:
+    name: test-component-library-vizro-core
```

I like how this name matches our usual scheme 👍 You shouldn't need to specify `name` though given that there's no matrix entry for this.

Let's discuss overall naming but I think e2e is probably good.

```suggestion
  test-e2e-component-library-vizro-core:
```